shotgun_api3/lib/mimetypes.py

```diff
@@ -115,7 +117,10 @@ def guess_type(self, url, strict=True):
         Optional `strict' argument when False adds a bunch of commonly found,
         but non-standard types.
         """
-        scheme, url = urllib.splittype(url)
+        parseResult = urlparse(url)
+        scheme = parseResult.scheme
+        url = parseResult.netloc + parseResult.path
```

Comment:
This doesn't give us exactly the same string as the previous implementation.
previous implementation:
url = "//host/path.ext"

current implementation (with the fix):
url = "host/path.ext"

The code should behave the same regardless.